sickbeard/scene_exceptions.py

```diff
@@ -320,7 +320,7 @@ def _xem_exceptions_fetcher():
             if not parsedJSON['data']: 
                 logger.log(u"No data returned from XEM when checking scene exceptions. Update failed for {0}".format
                             (sickbeard.indexerApi(indexer).name), logger.DEBUG)
-                continue
+                break
```

Comment:
I agree with you but @labrys said to break

```
[20:11:02] <Labrys> and allNames should always return a value as i expected
[20:11:22] <Labrys> so imo it should break if no value because it means xem is down
```